### PR TITLE
Add a pleasant Ahelp sound, allow use of BWOINK if needed

### PR DESCRIPTION
**Pull request by rhicora**
Inbound ahelps now play a pleasant chime for all admins, unless they've muted it.

Outbound PMs will either send the player a pleasant sound, or a BWOINK sound, depending on the admin's urgency preference. This preference defaults to "pleasant" but can be set to "loud." If the admin is in loud mode, remind them so by BWOINKing them too.

Staffwarns always play a BWOINK for the player because they did a bad.

### CODE REVIEW

**Comment by PrPleGoo** [comment]
"A sense of alarm, urgency, and conflict is instantly summoned"
That's the point

**Comment by Kurgis** [comment]
The sound/machines/ping.ogg file is used and referenced in dozens of machines. It is used by bots, borgs, kitchen appliances. I don't anyone tabbed out will notice or discern what the ping is when admin messaged.

If you want to replace the bwoink sound, replace it with a new unique sound file, which is not used by any other in-game object, item, machine, or structure.

**Comment by rhicora** [comment]
By Pipe's suggestion, I'll also make alarming-bwoink something admins can toggle on if they really need to get someone's attention.

File: code/game/world.dm [hunk 1]
```diff
@@ -372,7 +372,7 @@ var/world_topic_spam_protect_time = world.timeofday
 		C.received_irc_pm = world.time
 		C.irc_admin = input["sender"]
 
-		sound_to(C, 'sound/effects/adminhelp.ogg')
+		sound_to(C, 'sound/effects/pleasant.ogg')
```
Comment (rhicora) [inline code/game/world.dm]: PMs from IRC.

File: code/modules/admin/verbs/adminhelp.dm [hunk 2]
```diff
@@ -124,7 +124,7 @@ var/list/adminhelp_ignored_words = list("unknown","the","a","an","of","monkey","
 			if(X.is_afk())
 				admin_number_afk++
 			if(X.get_preference_value(/datum/client_preference/staff/play_adminhelp_ping) == GLOB.PREF_HEAR)
-				sound_to(X, 'sound/effects/adminhelp.ogg')
+				sound_to(X, 'sound/effects/pleasant.ogg')
```
Comment (rhicora) [inline code/modules/admin/verbs/adminhelp.dm]: Incoming Ahelps from players.

File: code/modules/admin/verbs/adminpm.dm [hunk 3]
```diff
@@ -140,10 +140,13 @@
...
 
-	//play the recieving admin the adminhelp sound (if they have them enabled)
+	//play the recieving player the adminhelp sound (if they have them enabled)
 	//non-admins shouldn't be able to disable this
 	if(C.get_preference_value(/datum/client_preference/staff/play_adminhelp_ping) == GLOB.PREF_HEAR)
-		sound_to(C, 'sound/effects/adminhelp.ogg')
+		// only do it if the sending admin has turned on loud bwoinking
+		if (src.get_preference_value(/datum/client_preference/staff/bwoink_urgency) == GLOB.PREF_LOUDLY)
+			sound_to(C, 'sound/effects/adminhelp.ogg') // GOT A MINUTE?
+			sound_to(src, 'sound/effects/adminhelp.ogg') // Let the admin know they're bwoinking!
```
Comment (rhicora) [inline code/modules/admin/verbs/adminpm.dm]: Outgoing ahelps to players.

File: code/modules/client/preference_setup/global/preference_datums.dm [hunk 4]
```diff
@@ -206,10 +208,15 @@ var/list/_client_preferences_by_type
...
 /datum/client_preference/staff/play_adminhelp_ping
-	description = "Adminhelps"
+	description = "Ahelp Ping"
 	key = "SOUND_ADMINHELP"
 	options = list(GLOB.PREF_HEAR, GLOB.PREF_SILENT)
 
+/datum/client_preference/staff/bwoink_urgency
+	description = "Admin PM Players"
+	key = "BWOINK_URGENCY"
+	options = list(GLOB.PREF_PLEASANTLY, GLOB.PREF_LOUDLY)
```
Comment (rhicora) [inline code/modules/client/preference_setup/global/preference_datums.dm]: Let admins set how urgent they want to be.

**Comment by rhicora** [comment]
Ready to go now!

**Comment by Zero-Bits** [comment]
Still looks good and passes Travis.

**Pull request merged**